ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/methods/PrivacyApiGroupJsonRpcMethods.java

```diff
@@ -42,6 +44,7 @@
   private final ProtocolSchedule<?> protocolSchedule;
   private final TransactionPool transactionPool;
   private final PrivacyParameters privacyParameters;
+  private final ChainHeadPrivateNonceProvider nonceProvider;
```

Comment:
This should probably be using the `PrivateNonceProvider` interface, right? The same thing applies for the getter method.